sig-list.md

```diff
@@ -32,7 +32,7 @@ Each group's material is in its subdirectory in this project.
 |------|-----------|------------|---------|----------|
 |[sig-compute](sig-compute/charter.md) |[sig/compute](https://github.com/kubevirt/kubevirt/labels/sig/compute) | <ol><li>[Itamar Holder](https://github.com/iholder101), Red Hat</li><li>[Jed Lejosne](https://github.com/jean-edouard), Red Hat</li></ol> |[Slack](kubevirt-dev)<br/> [Mailing List](https://groups.google.com/forum/#!forum/kubevirt-dev) |<ul></ul> |
 |[documentation](sig-documentation/charter.md) |[sig/documentation](https://github.com/kubevirt/kubevirt/labels/sig/documentation) |  | |<ul></ul> |
-|[storage](sig-storage/charter.md) |[sig/storage](https://github.com/kubevirt/kubevirt/labels/sig/storage) |  | |<ul></ul> |
+|[storage](sig-storage/charter.md) |[sig/storage](https://github.com/kubevirt/kubevirt/labels/sig/storage) | <ol><li>[Alice Frosi](https://github.com/alicefr), Red Hat</li><li>[Michael Henriksen](https://github.com/mhenriks), Red Hat</li></ol> | |<ul><li>KubeVirt Storage SIG Meeting: [ Thursday at 1:00 PM UTC (bimonthly) ](https://zoom.us/j/97050528531)</li></ul> |
```

Comment:
Aren't the meetings on Mondays?

Reply:
@aburdenthehand does it have to be? I'm just happy that these meetings are taking place now.

Reply:
Yes they are Mondays

Reply:
Sorry wrong copy paste